src/searching.jl

```diff
@@ -0,0 +1,41 @@
+using GeometryBasics
+using Delaunator
+
+
+function _prevedge(i::Integer)
+    (i-1) % 3 == 0 ? i+2 : i -1
+end 
+
+function findtriangle(t, p)
+  current = 1
+  start = current
+  n = 0
+
+  while true
+      next = Delaunator._nextedge(current)
+      pc = t.points[(t._triangles[current])]
```

Comment:
for various reasons, it's better to assign points = t.points outside the for loop. (or it was at some point.) Same with t._triangles. and t._halfedges...